recipe/meta.yaml

```diff
@@ -1,6 +1,6 @@
 {% set name = "apispec" %}
-{% set version = "0.27.1" %}
-{% set sha256 = "878e525c6f171afdb5ac99001c05a4fdc2d2ede373fe6c862f282940b2f9ab19" %}
+{% set version = "0.29" %}
```

Comment:
`0.29.0`

Reply:
Thanks for the review. Should be fixed now.